tensorflow/go/op/scope_test.go

```diff
@@ -95,6 +95,40 @@ func TestMultipleGeneratedOps(t *testing.T) {
 	}
 }
 
+func TestScopeWithGraph(t *testing.T) {
+	s1 := NewScope()
+
+	var hello interface{} = "hello"
+	tensor1, _ := tf.NewTensor(hello)
+	s1.AddOperation(tf.OpSpec{
```

Comment:
Any particular reason to avoid the the `Const` method used in other tests in this file?
So this could be as simple as:

```go
s1 := NewScope()
Const(s1, "hello")
graph, err := s1.Finalize()
if err != nil {
  t.Fatal(err)
}

s2 := NewScopeWithGraph(graph)
Const(s2, "world")
if err := s2.Err(); err != nil {
  t.Fatal(err)
}
```